### ISSUE TITLE
TF: Use TG Papertrail Add-on

### ISSUE THREAD

**Issue by shaunxp20** [Iridescent-CM/technovation-app#2777]
TF and TG have separate Papertrail add-ons, which means we have to login to separate accounts to access each of them. And it means we're paying for two add-ons.

We can have one Papertrail add-on that both TF and TG use.

> Reuse existing subscription: attach additional app(s) to an existing Papertrail subscription, like this. The subscription will still use the payment method and collaborators of the app which it was first activated on, but Heroku will also send logs from the additional app(s). All collaborators can view logs from all apps.

[https://documentation.solarwinds.com/en/Success_Center/papertrail/Content/kb/hosting-services/heroku.htm#multiple-heroku-apps](https://documentation.solarwinds.com/en/Success_Center/papertrail/Content/kb/hosting-services/heroku.htm#multiple-heroku-apps)

How to attach an add-on to another app:
[https://devcenter.heroku.com/articles/managing-add-ons#using-the-command-line-interface-attaching-an-add-on-to-another-app](https://devcenter.heroku.com/articles/managing-add-ons#using-the-command-line-interface-attaching-an-add-on-to-another-app)

Action:
* Attach TG Papertrail add-on to TF
* Transfer TF alerts (and other things we want to save) to TG
* Cancel TF add-on

Notes:
I downgraded the Papertrail plan for TG in #2775 to the 250 MB plan. The four week daily average for TF ls only 8.9 MB (with a peak less than 20 MB), so it should be able to handle it okay.

**Comment by shaunxp20** [comment]
@stenington When you get a chance, can you double check me on this one, see if it makes sense to do. Am I missing anything? Any extra configuration to be aware of?

> Transfer TF alerts (and other things we want to save) to TG

Is there anything worth saving in the TF Papertrail? Looks like all the alerts are disabled.

**Comment by stenington** [comment]
Yeah we used to have alerts in Slack for a couple of things, but we never took any action based on the alerting so I shut it down some time ago to avoid (or try to recover from?) alert fatigue. At this point if I was asked to diagnose a problem on TF and I wanted Papertrail alerts, I'd start fresh anyway so I don't think the old alerts need to be carried over. I would say you're fine to merge them.

**Comment by shaunxp20** [comment]
Unfortunately this one isn't possible at the moment because TG and TF have different account owners "info@" vs "services@".
`Add-ons may only be attached to apps with the same owner.`

We need to get the apps owned by one account before this can happen.

**Comment by shaunxp20** [comment]
This one is blocked until we combine the TF/TG accounts, I added #2791 for this.
